app/views/dit_landing_page/_training.html.erb

```diff
@@ -5,7 +5,7 @@
         <h2 class="govuk-heading-l">
           <%= t("dit_landing_page.training_section_title") %>
         </h2>
-        <p class="govuk-body"><%= t("dit_landing_page.training_section_description") %></p>
+        <p class="govuk-body"><%= sanitize(t("dit_landing_page.training_section_description", url: t("dit_landing_page.training_section_url"))) %></p>
```

Comment:
Could we embed the URL in the description instead? It's a bit surprising to see us passing it back through in code.